poetry.lock

```diff
@@ -174,6 +174,14 @@ tomli = {version = "*", optional = true, markers = "python_full_version <= \"3.1
 [package.extras]
 toml = ["tomli"]
 
+[[package]]
```

Comment:
I have copied from the main branch of strawberry-django-plus, and did poetry add --dev as you said.